plans/artifactory-pro/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Starting Artifactory Pro"
+
+export JAVA_HOME=$(hab pkg path core/server-jre)
+export ARTIFACTORY_HOME={{pkg.svc_var_path}}
+
+{{pkg.svc_var_path}}/bin/artifactory.sh 2>&1
```

Comment:
FYI I saw @adamhjk switched the `redis` plan from using a `run` hook to a line in `plan.sh`:
```bash
pkg_svc_run="bin/redis-server $pkg_svc_config_path/redis.config"
```